egs3/librispeech_100/asr1/conf/evaluate.yaml

```diff
@@ -0,0 +1,36 @@
+# Decoding configuration for LibriSpeech 100h
+defaults:
+  - default.yaml
+  - ../dataset.yaml
+
+exp_tag: template
+
+
+model:
+  _target_: espnet2.bin.asr_inference.Speech2Text
+  asr_train_config: ${exp_dir}/config.yaml
+  asr_model_file: ${exp_dir}/last.ckpt
+  beam_size: 1
+  ctc_weight: 1.0
+
+
+metrics:
+  - metric:
+      _target_: template.metrics.wer.WER
+      clean_types: whisper_basic
```

Comment:
We should not use whisper filtering for the LibriSpeech WER computation.
This will reach different results from the others.
People do not do any normalization for the normal Librispeech evaluation.